src/drafting/draftutil.js

```diff
@@ -36,41 +36,57 @@ export const getDrafterState = ({ draft, seatNumber, pickNumber = -1, stepNumber
   let amount = 0;
   let packNum = 0;
   let offset = 0;
-  for (; packNum < numPacks; packNum++) {
-    let done = false;
+
+  // loop through each pack
+  while (packNum < numPacks) {
     const curPackNum = packNum;
+
+    let done = false;
     packsWithCards = draft.initial_state.map((packsForSeat) => packsForSeat[curPackNum].cards.slice());
     pickNum = 0;
     packSize = packsWithCards[seatNum].length;
     offset = 0;
-    const steps = ourPacks[packNum].steps ?? defaultStepsForLength(ourPacks[packNum].cards.length);
+    const steps = ourPacks[curPackNum].steps ?? defaultStepsForLength(ourPacks[curPackNum].cards.length);
```

Comment:
I believe `curPackNum` is a totally redundant variable in this loop, given that `packNum` isn't being updated until the end of it.